pkgs/by-name/go/goobook/package.nix

```diff
@@ -1,46 +1,39 @@
 {
   lib,
-  buildPythonPackage,
   fetchFromGitLab,
-  pythonOlder,
   docutils,
   installShellFiles,
-  poetry-core,
-  google-api-python-client,
-  simplejson,
-  oauth2client,
-  setuptools,
-  pyxdg,
+  python3,
 }:
 
-buildPythonPackage rec {
+python3.pkgs.buildPythonApplication rec {
```

Comment:
wouldn't `python3Packages.buildPythonApplication` be better here?

Reply:
Because of cross-compiling?

Reply:
is that a thing? does cross-compiling not work with python3Packages?

Reply:
Cross-compilation doesn't work with `python3.pkgs` but I still prefer that because it let's you use `packageOverrides`.